oppia/oppia-android
# Fixes part of #1460: Json file updates

## Explanation


Fixes part of #1460 

This mainly updates the json files related to `topics`, `subtopics`, `skills` and `stories`. And does not update `explorations` and `questions`.

## How to test:
1. First proof that this implementation is correct is the fact that all domain layer test cases are passing with a minor change in one test case.
2. Before checking-in in this branch, check the entire app on develop, including concept-cards (described below), maybe take screenshots for better comparison later on.
3. Now run this app and check the entire app once again, like HomeScreen, Promoted Story Cards, Recently Played Stories (click on View All), 4 tabs in topic, exploration-player, question-player, revision-card in fractions (only 1st revision card will work), concept card (described below).
4. Checking concept card - for this, in manifest file, you will need to set `ConceptCardFragmentTestActivity` as launcher activity.

### Note: The file names have been changed to <id>.json to make code more general which will be very clear in next couple of PRs. But the main concept is that we will name all json files with `<id>.json` that way whenever we  want some data we can directly reference that file using the id.

## Checklist
